src/assembly/linux/main/install_cosmos_plugin.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+#
+# Copy COSMOS plugin configuration to argument directory
+# This will likely be moved to the Adamant repository
+# COSMOS install directory argument should be relative to the Linux example path
+# Ex. `./install_cosmos_plugin.sh cosmos-project` if COSMOS is adjacent to adamant_example
+
+cosmos_install_name=$1
+cosmos_install_dir=`realpath ../../../../../$cosmos_install_name`
+cosmos_plugin_dir=`realpath $cosmos_install_dir/openc3-cosmos-linux-example`
+adamant_example_plugin_dir=`realpath ../build/cosmos/plugin`
+adamant_protocol_dir=`realpath ../../../../../adamant/gnd/cosmos`
+echo cp $adamant_example_plugin_dir/linux_example_ccsds_cosmos_commands.txt $cosmos_plugin_dir/targets/LINUX_EXAMPLE/cmd_tlm/cmd.txt
+echo cp $adamant_example_plugin_dir/linux_example_ccsds_cosmos_telemetry.txt $cosmos_plugin_dir/targets/LINUX_EXAMPLE/cmd_tlm/tlm.txt
+echo cp $adamant_example_plugin_dir/linux_example_ccsds_cosmos_plugin.txt $cosmos_plugin_dir/plugin.txt
```

Comment:
Hard-coding "linux_" in this script makes it not reusable for other assemblies. We should get the assembly name by looking for a .assembly.yaml file in the directory. Then we can call this script to install a plugin for any Adamant assembly. The easiest way to do this might actually be to just pass the path to the assembly file as argument to this script. One more note, you should be able to use $ADAMANT_DIR to find the root of the adamant directory. That is defined once the Adamant environment is activated.

Reply:
Addressed in [cbf2c22](https://github.com/lasp/adamant_example/pull/18/commits/cbf2c22f7e1b6331126c7b070050bb46cfe1dcb1). This script now takes the assembly name as an argument, Readme is also updated to reflect this.